sokoban/src/App.tsx

```diff
@@ -41,7 +75,13 @@ function App() {
             <div
               className="cell"
               key={cellid}
-              style={{ width: 500 / newGameBoard[0].length, backgroundColor: `${colors[cell]}` }}
+              style={{
```

Comment:
Maybe to much inline styling?

Reply:
We did not added className, but if you want we will delete it. 

Row 44 is this for us:
          row.map((cell: string, cellid: number) => (
            <div           
              key={cellid}
              style={{
                width: 500 / newGameBoard[0].length,
                backgroundImage: `url(${backgoundImage[cell]})`,  ----- here is the cell information
                backgroundPosition: "center",
                backgroundSize: "cover",
                backgroundRepeat: "no-repeat",
              }}
            >
              {cell}
            </div>
          ))

I try to party styling:
const backgroundStyle = { width: 500 / newGameBoard[0].length, backgroundPosition: "center", backgroundSize: "cover", backgroundRepeat: "no-repeat" }

then in the return:
   style={backgroundStyle { backgoundImage: `url(${backgoundImage[cell]})` }}
But it does not work for me. 

I can add setProperty to have a partial styling with javascript and leave the cellinfo styling in the return but it seems more ugly.

I appreciate any solution.

Reply:
ja. little bit extra inline styling but , as Judi said we tried many other options but did not work, if any one has better solution then please share.